pkg/deploy/dev-workspace/dev_workspace.go

```
@@ -55,96 +50,103 @@ var (
+		return reconcile.Result{Requeue: true}, false, err
 	}
 	if !isCreated {
 		namespace := &corev1.Namespace{}
-		err := deployContext.ClusterAPI.NonCachingClient.Get(context.TODO(), types.NamespacedName{Name: DevWorkspaceNamespace}, namespace)
+		err := ctx.ClusterAPI.NonCachingClient.Get(context.TODO(), types.NamespacedName{Name: DevWorkspaceNamespace}, namespace)
 		if err != nil {
-			return false, err
+			return reconcile.Result{Requeue: true}, false, err
 		}
 
 		namespaceOwnershipAnnotation := namespace.GetAnnotations()[deploy.CheEclipseOrgNamespace]
 		if namespaceOwnershipAnnotation == "" {
 			// don't manage DWO if namespace is create by someone else not but not Che Operator
-			return true, err
+			return reconcile.Result{}, true, nil
 		}
 
 		// if DWO is managed by another Che, check if we should take control under it after possible removal
-		if namespaceOwnershipAnnotation != deployContext.CheCluster.Namespace {
-			isOnlyOneOperatorManagesDWResources, err := isOnlyOneOperatorManagesDWResources(deployContext)
+		if namespaceOwnershipAnnotation != ctx.CheCluster.Namespace {
+			isOnlyOneOperatorManagesDWResources, err := isOnlyOneOperatorManagesDWResources(ctx)
 			if err != nil {
-				return false, err
+				return reconcile.Result{Requeue: true}, false, err
 			}
 			if !isOnlyOneOperatorManagesDWResources {
 				// Don't take a control over DWO if CheCluster in another CR is handling it
-				return true, nil
+				return reconcile.Result{}, true, nil
 			}
-			namespace.GetAnnotations()[deploy.CheEclipseOrgNamespace] = deployContext.CheCluster.Namespace
-			_, err = deploy.Sync(deployContext, namespace)
+			namespace.GetAnnotations()[deploy.CheEclipseOrgNamespace] = ctx.CheCluster.Namespace
+			_, err = deploy.Sync(ctx, namespace)
 			if err != nil {
-				return false, err
+				return reconcile.Result{Requeue: true}, false, err
 			}
 		}
 	}
 
 	// check if DW exists on the cluster
 	devWorkspaceWebHookExistedBeforeSync, err := deploy.Get(
```

I think, here is we ignore error.